account-kit/rn-signer/example/package.json

```diff
@@ -3,22 +3,35 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
+    "android": "react-native run-android && yarn start:redirect-server",
     "ios": "react-native run-ios",
     "start": "react-native start",
+    "start:redirect-server": "tsx ./redirect-server/index.ts",
     "build:android": "react-native build-android --extra-params \"--no-daemon --console=plain -PreactNativeArchitectures=arm64-v8a\"",
     "build:ios": "react-native build-ios --scheme ReactNativeSignerExample --mode Debug --extra-params \"-sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO\""
   },
   "dependencies": {
     "@account-kit/signer": "^4.3.0",
+    "@react-navigation/native": "7.0.3",
+    "@react-navigation/native-stack": "7.1.0",
+    "dotenv": "^16.4.5",
+    "express": "^4.21.1",
     "react": "18.3.1",
     "react-native": "0.76.1",
-    "react-native-config": "^1.5.3",
-    "react-native-mmkv": "^3.1.0",
+    "react-native-config": "1.5.3",
+    "react-native-gesture-handler": "2.21.2",
+    "react-native-mmkv": "3.1.0",
```

Comment:
we should add this as a peer dependency to the `rn-signer` package itself if it's not already just so that package managers call this out to folks using our library